cvjude/Banka
# [Feature #164698027] Create the Signin page

#### What does this PR do?
Creates the Sign in page

#### Description of Task to be completed?
Create the Sign in page and make it responsive

#### How should this be manually tested?
This can be tested manually be visiting the site hosted on gh-pages
http://cvjude.github.io/Banka/UI/index.html, When on this site click on the sign in option from the nav bar and You should be navigated to the sign in page/section.

#### What are the relevant pivotal tracker stories?
<a href = "https://www.pivotaltracker.com/story/show/164788942">#164788942</a>
